compiler/src/iree/compiler/Codegen/LLVMCPU/test/pipeline_ssve_tests.mlir

```diff
@@ -27,27 +26,6 @@ module {
 // CHECK: func.func @dispatch() 
 // CHECK-SAME: arm_locally_streaming
 
-// -----
```

Comment:
Its OK to drop this, but we dont need to I think?

Reply:
Not sure how valuable this is without `CPUDoubleTilingPeelingExpert`. This file makes sure that that the streaming mode is enabled for the supported lowering experts (see `// CHECK-SAME: arm_locally_streaming`).

Without `CPUDoubleTilingPeelingExpert`, I can only test `CPUDoubleTilingExpert` with the `enable_loop_peeling` attribute. But is it _that_ different to testing `CPUDoubleTilingExpert`?